code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -74,9 +72,11 @@
 	if(tally > 0 && (CE_SPEEDBOOST in chem_effects))
 		tally = max(0, tally-3)
 
-	return (tally+config.human_delay)
-
+	var/turf/T = get_turf(src)
+	if(T && T.movement_cost)

```

Comment:
`turf/movement_cost` can never be a variable that can't be added to an already be added to a number. So just `if (T)` should suffice, no?

Reply:
what

Reply:
`if (T && T.movement_cost) tally += T.movement_cost` is pointless because `T.movement_cost` will always be a number. You might as well just `if (T) tally += T.movement_cost`.